workers/data_refinery_workers/processors/qn_reference.R

```diff
@@ -0,0 +1,59 @@
+###
+# qn_reference
+# Alex's Lemonad Stand Foundation
+# Childhood Cancer Data Lab
+#
+###
+
+#######################
+# The command interface!
+#######################
+
+library("optparse")
+library(data.table)
+library("preprocessCore")
+
+option_list = list(
+  make_option(c("-i", "--inputFile"), type="character", default="", 
+              help="inputFile", metavar="character"),
+  make_option(c("-o", "--outputFile"), type="character", default="", 
+              help="outputFile", metavar="character")
+); 
+
+opt_parser = OptionParser(option_list=option_list);
+opt = parse_args(opt_parser);
+
+filePath <- opt$inputFile
+outFilePath <- opt$outputFile
+
+# Get and use this DB
+message("Loading...")
+source("http://bioconductor.org/biocLite.R")
+
+# Read the data file
+message("Reading data file...")
+suppressWarnings(data <- fread(filePath, 
+					stringsAsFactors=FALSE, 
+					sep="\t", header=TRUE, 
+					autostart=10, 
+					data.table=FALSE, 
+					check.names=FALSE, 
+					fill=TRUE, 
+					na.strings="", 
+					showProgress=FALSE)
+				)
+
+message(data)
+
+# Create the target result
+data_matrix <- data.matrix(data, rownames.force = NA)
+data_matrix<-data_matrix[,-1] # We don't want the index column
```

Comment:
Will the reference data that goes through our smasher always have some kind of identifier in the first column? Since this was developed with the toy data I posted, that had row indices in the first column and would cause an issue. Something to keep an eye on when working with real data.